api/prd/lov/data/lov.mock.data.ts

```diff
@@ -160,6 +160,114 @@ export const DEFAULT_HEARING_CANCEL_REF: LovRefDataModel[] = [
   },
 ];
 
+export const DEFAULT_HEARING_ACTUAL_COMPLETE_REF: LovRefDataModel[] = [
+  {
+    key: 'decisionInFavourOfAppellant',
+    value_en: 'Decision in favour of Appellant',
+    value_cy: '',
+    hintText_EN: 'Decision in favour of Appellant',
+    hintTextCY: '',
+    order: 1,
+    parentKey: null,
+  },
+  {
+    key: 'decisionReserved',
+    value_en: 'Decision Reserved',
+    value_cy: '',
+    hintText_EN: 'Decision Reserved',
+    hintTextCY: '',
+    order: 2,
+    parentKey: null,
+  },
+  {
+    key: 'decisionRevisedAgainstAppeallant',
+    value_en: 'Decision Revised Against Appeallant',
+    value_cy: '',
+    hintText_EN: 'Decision Revised Against Appeallant',
+    hintTextCY: '',
+    order: 4,
+    parentKey: null,
+  },
+];
+
+export const DEFAULT_HEARING_ACTUAL_ADJOURN_REF: LovRefDataModel[] = [
+  {
+    key: 'postponedDueToExclusions',
+    value_en: 'Postponed, due to Exclusions',
+    value_cy: '',
+    hintText_EN: 'Postponed, due to Exclusions',
+    hintTextCY: '',
+    order: 1,
+    parentKey: null,
+  },
+  {
+    key: 'postponedDueToOtherReasons',
+    value_en: 'Postponed, due to Other Reasons',
+    value_cy: '',
+    hintText_EN: 'Postponed, due to Other Reasons',
+    hintTextCY: '',
+    order: 2,
+    parentKey: null,
+  },
+  {
+    key: 'postponedIncompleteTribunal',
+    value_en: 'Postponed, Incomplete Tribunal',
+    value_cy: '',
+    hintText_EN: 'Postponed, Incomplete Tribunal',
+    hintTextCY: '',
+    order: 4,
+    parentKey: null,
+  },
+  {
+    key: 'postponedNoReasonGiven',
+    value_en: 'Postponed, No Reason Given',
+    value_cy: '',
+    hintText_EN: 'Postponed, No Reason Given',
+    hintTextCY: '',
+    order: 4,
+    parentKey: null,
+  },
+  {
+    key: 'postponedOtherPartyUnableToAttend',
+    value_en: 'Postponed, Other Party unable to attend',
+    value_cy: '',
+    hintText_EN: 'Postponed, Other Party unable to attend',
+    hintTextCY: '',
+    order: 4,
+    parentKey: null,
+  },
+];
```

Comment:
Could you confirm with BA and see if this is still matched with requirement?
Are you refering to this prototype https://ucd-xui2-prototype.herokuapp.com/hearing-actuals/version3/actuals-details or others?

Reply:
@rayliangatos thanks for bringing this up in today's stand-up. I have now removed this code and made necessary changes as dropdown is not required for the first radio button option.